requirements.txt

```diff
@@ -6,7 +6,11 @@
 colorama
 cray-product-catalog
 PyYAML
+requests==2.25.1
 requests-retry-session
+kubernetes==26.1.0
+python-keycloak==0.27.1
+urllib3==1.26.20
```

Comment:
The `urllib3` package is already available in the virtual environment from the system Python packages. See later in this file.
```suggestion
```
Is there a reason you need this specific version of the package?

Reply:
yes the keycloak version needs this version of urllib3

Reply:
That doesn't seem to be the case from my testing -- are you sure?